src/config/dev.js

```diff
@@ -4,7 +4,7 @@ import baseConfig from './base';
 
 let config = {
   appEnv: 'dev',  // feel free to remove the appEnv property here
-  apiBase: 'http://localhost:8888/v0/',
+  apiBase: 'https://api.openframe.io/v0/',
```

Comment:
@jvolker Notice this — this points the local dev web app at the production api. If/when we get the API server running locally again we should switch this default back. 

Might be better long-term to move these settings to env vars.